db/seeds.rb

```diff
@@ -23,6 +23,7 @@
     }
   ) do |news_article|
     news_article.name = "Business News"
+    news_article.release_date = Date.today
```

Comment:
Probably best to make this an update call afterwards because this block is only called if the article doesn't exist. We want to fix the release date for anyone who's already run seeds.